requirements.txt

```diff
@@ -3,7 +3,7 @@
 google-api-python-client~=1.6
 matplotlib~=2.2
 networkx~=2.1
-numpy~=1.12
+numpy~=1.16
```

Comment:
Took this opportunity because tf-nightly is not running with older versions of numpy (got a `ModuleNotFoundError: No module named 'numpy.core._multiarray_umath'` ) when trying to import.